README.md

```diff
@@ -125,6 +125,16 @@ Run your tests with a command like this:
 npm run codecov
 ```
 
+### [Github Actions](https://github.com/codecov/codecov-action)
+
+```yaml
+- name: Codecov
+  uses: codecov/codecov-action@v1
```

Comment:
```suggestion
  uses: codecov/codecov-action@v2
```
See https://about.codecov.io/blog/introducing-codecovs-new-uploader/ for detail.